tools/tests/unit/test_spice_utils.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import os
+from datetime import datetime
+from pathlib import Path
+
+import pytest
+import spiceypy as spice
+
+from tools.spice.spice_utils import (
+    SpiceKernelManager,
+    ls_attitude_coverage,
+    ls_kernels,
+    ls_spice_constants,
+)
+
+
+@pytest.fixture()
+def kernel_object():
+    """Data for decom"""
+    directory = Path(__file__).parent.parent / "test_data" / "spice"
+    extensions = [".bpc", ".bsp", ".ti", ".tf", ".tls", ".tsc"]
+    kernels = SpiceKernelManager(directory, extensions)
+    return kernels
+
+
+def test_furnsh(kernel_object):
+    kernel_object.furnsh()
+    n_files_loaded = spice.ktotal("ALL")
+
+    extensions = [".bpc", ".bsp", ".ti", ".tf", ".tls", ".tsc"]
+
+    files = [
+        file
+        for file in os.listdir(kernel_object.kernel_path)
+        if file.endswith(tuple(extensions))
+    ]
+
+    n_test_files = len(files)
+
+    assert n_files_loaded == n_test_files
+
+
+def test_clear(kernel_object):
+    # Ensure some kernels are loaded first
+    kernel_object.furnsh()
+    n_files_before_clear = spice.ktotal("ALL")
+    assert n_files_before_clear > 0
+
+    # Clear loaded kernels and verify
+    kernel_object.clear()
+    n_files_loaded_after_clear = spice.ktotal("ALL")
+    assert n_files_loaded_after_clear == 0
+
+
+def test_furnsh_type(caplog):
```

Comment:
I'm a bit confused as to what this test is checking for. It looks more like it is making sure that an invalid kernel type is resulting in an expected error message?  If so, might want to rename the test to something like `test_invalid_kernel`. It would also be helpful to include a docstring.